ui/app/common/photo-capture/directives/capturePhoto.js

```diff
@@ -130,7 +130,7 @@ angular.module('bahmni.common.photoCapture')
             };
 
             scope.uploadImage = function () {
-                if (this.files && this.files[0]) {
+                if (this.files && this.files[0] && this.files[0].size <= 500000) {
```

Comment:
Wouldn't it be better to externalize the size as configuration?

Reply:
Sure, we will discuss and update accordingly.

Reply:
As we discussed we have externalized the size(**imageUploadSize**) as configuration. Also to not choke up the server capacity we have added maximum size(**maxImageUploadSize**) for the upload.